src/main/java/app/entities/Cart.java

```diff
@@ -0,0 +1,38 @@
+package app.entities;
+
+import java.util.List;
+
+public class Cart {
+
+    private List<Orderline> orderlineList;
+
+
+    public Cart(List<Orderline> orderlineList) {
+        this.orderlineList = orderlineList;
+
+    }
+
+    public double getTotalPriceOfCart(){
```

Comment:
The total value of an orderline is never calculated and is allways assumed to be known